config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_certificatemanagerdnsauthorizations.certificatemanager.cnrm.cloud.google.com.yaml

```diff
@@ -308,9 +341,3 @@ spec:
     storage: true
```

Comment:
Could you try to:
1. Install the old CRD and create a CertificateManagerDNSAuthorization resource.
2. Upgrade to the new CRD.
3. Verify whether the upgrade works and whether the existing CertificateManagerDNSAuthorization successfully reconciled based on the new schema?

The reason I asked here is that we've never switched the storage version from v1alpha1 to v1beta1 so we probably want to be extra careful.

Reply:
I tested by following these steps:

1. Checkout to master
2. Apply the existing crd `kubectl apply -f config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_certificatemanagerdnsauthorizations.certificatemanager.cnrm.cloud.google.com.yaml; make deploy-controller && kubectl delete pods --namespace cnrm-system --all`
3. Apply sample DNS Authorization `kubectl apply -f config/samples/resources/certificatemanagerdnsauthorization/certificatamanager_v1beta1_certificatemanagerdnsauthorization.yaml`
4. Check into my branch with new change and run step 2 again with an extra log line to confirm that the new controller is deployed.
5. Wait a bit and confirm that the resource is still good.
6. Make a change in the resource and apply again to force reconciliation.
7. The resource is updated and ready.

Reply:
Awesome, thank you!